Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -193,7 +193,7 @@ uplink-syndicate-segway-crate-desc = Be an enemy of the corporation, in style!
 uplink-revolver-cap-gun-name = Cap Gun
 uplink-revolver-cap-gun-desc = Looks almost like the real thing! Ages 8 and up.
 
-uplink-syndicate-stamp-name = Syndicate Rubber Btamp
+uplink-syndicate-stamp-name = Syndicate Rubber Stamp # In loving memory, Btamp, ???-2/1/23
```

Comment:
what

Reply:
oh wait btamp meme nvm

Reply:
spotted a typo there when I was making the ftl entry

Reply:
at least format the date correctly.